usaspending_api/disaster/management/sql/disaster_covid19_file_d1_awards.sql

```diff
@@ -0,0 +1,304 @@
+    "transaction_fpds"."contracting_officers_deter" AS "contracting_officers_determination_of_business_size_code",
+    "transaction_fpds"."emerging_small_business" AS "emerging_small_business",
+    "transaction_fpds"."community_developed_corpor" AS "community_developed_corporation_owned_firm",
+    "transaction_fpds"."labor_surplus_area_firm" AS "labor_surplus_area_firm",
+    "transaction_fpds"."us_federal_government" AS "us_federal_government",
+    "transaction_fpds"."federally_funded_research" AS "federally_funded_research_and_development_corp",
+    "transaction_fpds"."federal_agency" AS "federal_agency",
+    "transaction_fpds"."us_state_government" AS "us_state_government",
+    "transaction_fpds"."us_local_government" AS "us_local_government",
+    "transaction_fpds"."city_local_government" AS "city_local_government",
+    "transaction_fpds"."county_local_government" AS "county_local_government",
+    "transaction_fpds"."inter_municipal_local_gove" AS "inter_municipal_local_government",
+    "transaction_fpds"."local_government_owned" AS "local_government_owned",
+    "transaction_fpds"."municipality_local_governm" AS "municipality_local_government",
+    "transaction_fpds"."school_district_local_gove" AS "school_district_local_government",
+    "transaction_fpds"."township_local_government" AS "township_local_government",
+    "transaction_fpds"."us_tribal_government" AS "us_tribal_government",
+    "transaction_fpds"."foreign_government" AS "foreign_government",
+    "transaction_fpds"."organizational_type" AS "organizational_type",
+    "transaction_fpds"."corporate_entity_not_tax_e" AS "corporate_entity_not_tax_exempt",
+    "transaction_fpds"."corporate_entity_tax_exemp" AS "corporate_entity_tax_exempt",
+    "transaction_fpds"."partnership_or_limited_lia" AS "partnership_or_limited_liability_partnership",
+    "transaction_fpds"."sole_proprietorship" AS "sole_proprietorship",
+    "transaction_fpds"."small_agricultural_coopera" AS "small_agricultural_cooperative",
+    "transaction_fpds"."international_organization" AS "international_organization",
+    "transaction_fpds"."us_government_entity" AS "us_government_entity",
+    "transaction_fpds"."community_development_corp" AS "community_development_corporation",
+    "transaction_fpds"."domestic_shelter" AS "domestic_shelter",
+    "transaction_fpds"."educational_institution" AS "educational_institution",
+    "transaction_fpds"."foundation" AS "foundation",
+    "transaction_fpds"."hospital_flag" AS "hospital_flag",
+    "transaction_fpds"."manufacturer_of_goods" AS "manufacturer_of_goods",
+    "transaction_fpds"."veterinary_hospital" AS "veterinary_hospital",
+    "transaction_fpds"."hispanic_servicing_institu" AS "hispanic_servicing_institution",
+    "transaction_fpds"."contracts" AS "receives_contracts",
+    "transaction_fpds"."grants" AS "receives_financial_assistance",
```

Comment:
I don't see "receives_financial_assistance" on the data dictionary crosswalk. Do you have it on good word this translation is correct?

Reply:
That is the alias for the database column, which come from Ross. I can't speak to the Data Dictionary other than it is most likely outdated.

This field isn't new for COVID-19 data as these headers have been around for some time.